### ISSUE TITLE
1.4 What we plan to do and why

### ISSUE THREAD

**Issue by karthik** [si2-urssi/plan#44]
_The conceptualization phase with its survey, workshops, and ethnographic studies elucidated the need in the community for different components of a potential implementation of URSSI. We identified four areas for supporting the research software community - to accelerate science for diverse research domains as well as software engineering as a research area in its own right._

> Section 1.4, into paragraph: you could possibly add something to explain that the problem you’re dealing with is multi-faceted with lots of interdependencies, so it absolutely demands a multi-faceted solution, and to fund anything but that multi-faceted solution would lead to failure. We’ve had some concerns before that funders have wanted us to drop a facet of the Institute’s work, so we’ve been clear that removing a single spoke risks breaking the wheel.

**Comment by danielskatz** [comment]
This is part of the reason I think the mapping idea discussed in https://github.com/si2-urssi/plan/issues/39 is problematic

**Comment by karthik** [comment]
Isn' this counter to our thinking and discussions? We are not proposing an all or nothing approach. If for some reason the whole hog cannot be funded, there is still value in pursuing some of the spokes.

**Comment by danielskatz** [comment]
agreed, but the spokes are not directly tied to a single goal

**Comment by karthik** [comment]
The single goal here is the mission.  Education trains people. Incubators help them develop software with a higher likelihood of achieving sustainability. Policy advocates for the advancement of these people and community helps them coalesce. In our case some missing spokes wont derail the mission till later (e.g. slow impact like policy) but that doesn't mean we can't push forward with the other spokes.

> **Our mission is to improve the recognition, development, and use of software for a more sustainable research enterprise.** We achieve this mission through collaboratively developing education, outreach, and software services that emphasize open, transparent reproducible, and cooperative practices. URSSI is an institute for software expertise as well as a social infrastructure that promotes an inclusive and diverse community of research software engineers, maintainers, contributors, and users.

**Comment by danielskatz** [comment]
Do you have a suggestion for resolving this issue?

**Comment by karthik** [comment]
I agree with the first part of Simon's assessment:

_You could possibly add something to explain that the problem you’re dealing with is multi-faceted with lots of interdependencies_

This part is very obvious. As you see in 3.3, everything is connected. 

![image](https://user-images.githubusercontent.com/138494/85061963-8bdf5b80-b15c-11ea-9375-12860fadd5c8.png)


I don't agree with the second bit

 _so it absolutely demands a multi-faceted solution, and to fund anything but that multi-faceted solution would lead to failure._

**Comment by danielskatz** [comment]
I agree with you on the first point, but on the second, I'm in between you and Simon.  To me, a single-facet action would not lead to failure, but it also wouldn't be as effective a a multi-faceted action. I think that's part of the argument we are making for URSSI, rather than going off and proposing an incubator by itself, for example. There are links between activities that make them better together than they would be alone.

**Comment by karthik** [comment]
Very good. I see your point. I think it's a good one to articulate.

**Comment by SimonHettrick** [comment]
I hope it's okay if I clarify my original point!

For me, the argument relies on the difference between what is needed and what you'll accept.

I think you need all the different activities you described. Drop one, and I believe it will have a detrimental effect on your level of success. In the UK at least, funders like to cut costs by lopping off one or more of your activities, because there's a tendency to see all problems as infinitely divisible. I proposed the "multi-facted problem requires multi-facted solution" approach to try and ward off this divisible thinking. We used this argument for all phases of the SSI's funding (and, in response to an unforeseen drop in funding, we even accepted a 1.5 year cut to the duration of phase 2 rather than drop any of our activities).

However, in practice, we all know it's better to do something rather than nothing. So I'd set up the multi-faceted argument, then fight off any attempt to cut activities until its obvious you're not going to win. At that point, you accept a stripped down version of URSSI, because it's still going to make a big impact (just not as big...).

**Comment by danielskatz** [comment]
That matches what I was thinking but perhaps not explaining as well.
